tests/test_chronos_tools.py

```diff
@@ -93,50 +93,64 @@ def test_chronos_config_object_no_user(self):
         fake_json_contents = {
             'password': 'fake_password',
         }
-        fake_config = chronos_tools.ChronosConfig(fake_json_contents, 'fake_path')
+        fake_config = chronos_tools.ChronosConfig(fake_json_contents)
         with raises(chronos_tools.ChronosNotConfigured):
             fake_config.get_username()
 
     def test_chronos_config_object_no_password(self):
         fake_json_contents = {
             'user': 'fake_user',
         }
-        fake_config = chronos_tools.ChronosConfig(fake_json_contents, 'fake_path')
+        fake_config = chronos_tools.ChronosConfig(fake_json_contents)
         with raises(chronos_tools.ChronosNotConfigured):
             fake_config.get_password()
 
     def test_chronos_config_object_no_url(self):
         fake_json_contents = {
             'user': 'fake_user',
         }
-        fake_config = chronos_tools.ChronosConfig(fake_json_contents, 'fake_path')
+        fake_config = chronos_tools.ChronosConfig(fake_json_contents)
         with raises(chronos_tools.ChronosNotConfigured):
             fake_config.get_url()
 
     def test_load_chronos_config_good(self):
         expected = {'foo': 'bar'}
+        from_file = {'chronos_config': {'foo': 'bar'}}
         file_mock = mock.MagicMock(spec=file)
         with contextlib.nested(
-            mock.patch('paasta_tools.chronos_tools.open', create=True, return_value=file_mock),
-            mock.patch('json.load', autospec=True, return_value=expected)
+            mock.patch('paasta_tools.utils.open', create=True, return_value=file_mock),
+            mock.patch('json.load', autospec=True, return_value=from_file),
+            mock.patch('os.path.isdir', autospec=True, return_value=True),
+            mock.patch('os.access', autospec=True, return_value=True),
+            mock.patch('paasta_tools.utils.get_readable_files_in_glob', autospec=True,
+                       return_value=['/some/fake/dir/some_file.json']),
```

Comment:
I think I'd just mock out utils.load_system_paasta_config instead of all of these -- we test the behavior of load_system_paasta_config elsewhere, we can just trust it to work here.